vendor/libxml/libxml.gyp

```diff
@@ -1,56 +0,0 @@
-{
```

Comment:
why does this file need to be removed? This built on windows just fine previously. Can we just alter something in this file instead of copying it to the root gyp file?